drivers/watchdog/wdt_nrfx.c

```diff
@@ -141,6 +147,10 @@ static int wdt_nrf_feed(const struct device *dev, int channel_id)
 	if ((channel_id >= data->m_allocated_channels) || (channel_id < 0)) {
 		return -EINVAL;
 	}
+	if (data->enabled == false) {
```

Comment:
Why not just `if (!data->enabled) {`?

Reply:
There was no other pattern in the file to follow, they are equivalent, and the other checks in the file seem to be explicit
`if (cfg->callback != NULL)`

Reply:
> There was no other bool check pattern in the file to follow

Yes, there was no such pattern, but in `wdt_nrf_install_timeout()` you've added `if (data->enabled)` (which IMO looks much better than `if (data->enabled != false)`, so why not follow that? It's just a detail, I'm not going to insist.